Activities/Java/UiPath.Java.Activities.ViewModels/Resources/ActivitiesMetadataWindows.json

```
@@ -0,0 +1,48 @@
+{
+  "orderedCategoryDisplayNameKeys": [],

```

```suggestion
  "activities": [],

```

I think this is intended for property level metadata when listing the properties of activity.

---

I will remove it